app/components/utils/manage-alerts/component.js

```diff
@@ -15,8 +21,8 @@ export default Component.extend({
 
   title: getCachedProperty('title'),
   message: getCachedProperty('message'),
-  beginDate: getCachedProperty('beginDate'),
-  endDate: getCachedProperty('endDate'),
+  beginDate: getCachedProperty('beginDate') || new Date(), // eslint-disable-line ember/avoid-leaking-state-in-ember-objects
+  endDate: getCachedProperty('endDate') || new Date(), // eslint-disable-line ember/avoid-leaking-state-in-ember-objects
```

Comment:
Can't we use moment here instead?

Reply:
A bit further up I found out more about this. I would use moment and format it to a date string. The Date had me really confused. I'm curious to find out the motivation/reason behind this.